spec/dummy/bin/setup

```diff
@@ -20,12 +20,14 @@ chdir APP_ROOT do
   # Install JavaScript dependencies if using Yarn
   # system('bin/yarn')
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?('config/database.yml')
-  #   cp 'config/database.yml.sample', 'config/database.yml'
-  # end
+  puts "\n== Copying sample files =="
+  unless File.exist?('config/database.yml')
+    cp 'config/database.sample.yml', 'config/database.yml'
+  end
 
   puts "\n== Preparing database =="
+  system! 'bin/rails db:create'
+  system! 'bin/rails db:migrate'
   system! 'bin/rails db:setup'
```

Comment:
What does `db:setup` do?

Reply:
apparently, the 2 above it which I can drop.